thavalon-webapp/src/components/Login.tsx

```diff
@@ -69,32 +72,19 @@ export function Login(props: LoginProps) {
             className="Modal"
             overlayClassName="Overlay"
         >
-            <h2 className="modalHeader">Log In</h2>
-            <form onSubmit={handleSubmit(OnSubmit)}>
-                <input
-                    type="email"
-                    placeholder="Email"
-                    name="email"
-                    ref={register({
-                        required: true, maxLength: 80, pattern: {
-                            value: /^\S+@\S+\.\S+$/i,
-                            message: "Invalid email address."
-                        }
-                    })} />
-                {errors.email && <span className="errorMsg">{errors.email.message}</span>}
-                <br />
-                <input
-                    type="password"
-                    placeholder="Password"
-                    name="password"
-                    ref={register({required: true, minLength: 8})} />
-                {errors.password && <span className="errorMsg">Password required.</span>}
-                <br />
-                <Link to="/register" className="formLink">Create Account?</Link>
-                <br />
-                <input type="submit" disabled={disable} value="Log In" />
-                <span className="errorMsg">{formErrorMsg}</span>
-            </form>
+            <div className="modalContainer">
+                <h2 className="modalHeader">Log In</h2>
+                <hr />
+                <form onSubmit={handleSubmit(OnSubmit)}>
+                    <InputElement formRef={register} type="email" label="Email Address" name="email" required={true} minLength={0} />

```

Comment:
Should `minLength` be greater than 0 here?

Reply:
Really I should make it optional, don't think there's a minimum on db side @theadd336 can you confirm if db has a minimum?

Reply:
No minimum on the db for length at this time.